src/components/PickerMonth.vue

```diff
@@ -3,14 +3,14 @@
     <slot name="beforeCalendarHeader"></slot>
     <header>
       <span
-        @click="previousYear"
+        @click="isRtl ? nextYear() : previousYear()"
         class="prev"
-        :class="{ 'disabled' : isPreviousYearDisabled(pageTimestamp) }">&lt;</span>
+        :class="{ 'disabled' : isRtl ? isNextYearDisabled(pageTimestamp) : isPreviousYearDisabled(pageTimestamp) }">&lt;</span>
```

Comment:
Could we clean up this inline logic to a computed property?
maybe something like `isLeftNavDisabled` & `isRightNavDisabled`

Reply:
Yeah that's true it's like that there - Ok, I'll clean these up then.